modules/objc/examples/code_snippets/GettingStarted.m

```diff
@@ -43,27 +55,29 @@ - (void)viewDidLoad {
     // Create a query to fetch documents of type SDK.
     CBLQueryExpression *type = [[CBLQueryExpression property:@"type"] equalTo:[CBLQueryExpression string:@"SDK"]];
     CBLQuery *query = [CBLQueryBuilder select:@[[CBLQuerySelectResult all]]
-                                          from:[CBLQueryDataSource database:database]
+                                          from:[CBLQueryDataSource collection: collection]
                                          where:type];
 
     // Run the query
     CBLQueryResultSet *result = [query execute:&error];
     NSLog(@"Number of rows :: %lu", (unsigned long)[[result allResults] count]);
 
-    if (testReplication) {
+    if (replication) {
```

Comment:
Why we need this if here in the example code?

Reply:
explained below -> to guard against copy-paste-run and what not because repl errors on run as the URL endpoint is a placeholder and we dont instruct anywhere that that should be changed. An if statement is language agnostic and easy to view/read.